src/monitor.cc

```diff
@@ -404,6 +404,8 @@ void NodeMonitor::setStatistics() {
         Nan::GetHeapStatistics(&v8stats);
 
         stats_.pmem_ = (v8stats.used_heap_size() / (double) v8stats.total_heap_size());
+        stats_.usedheap_ = v8stats.used_heap_size();
+        stats_.totalheap_ = v8stats.total_heap_size();
```

Comment:
oh yes. that's probably a much better idea. I've updated the code to not do the extra call.